install/kubernetes/cilium/files/nodeinit/prestop.bash

```diff
@@ -46,7 +46,7 @@ echo "Restarting the kubelet"
 systemctl restart kubelet
 {{- end }}
 
-{{- if (and .Values.gke.enabled (or .Values.masquerade .Values.gke.disableDefaultSnat))}}
+{{- if (and .Values.gke.enabled (or .Values.enableIPv4Masquerade .Values.gke.disableDefaultSnat))}}
```

Comment:
while you are here, do you know where .Values.gke.disableDefaultSnat comes from ?

Reply:
I had to go a bit back in time to find it  — [`v1.8`](https://github.com/cilium/cilium/blob/v1.8/install/kubernetes/cilium/values.yaml#L448), it seems. It's absent in `v1.9` and later. I wonder if it can be removed too. 🤔

Reply:
For posterity, the `gke. disableDefaultSnat` value was introduced here: https://github.com/cilium/cilium/pull/12952

When true, it instructs nodeinit to remove the default IP masq entries that GKE puts into the route table.  It is redundant if `enableIPv4Masquerade` is true.